src/scripts/interface/dialogManager.ts

```
@@ -4,13 +4,22 @@ import { Elements } from 'interface/elements';
   private static readonly restartDialog = Elements.findById<HTMLDialogElement>(ElementIds.restartDialog);
   private static readonly dropdownDialog = Elements.findById<HTMLDialogElement>(ElementIds.dropdownDialog);
+  private static readonly settingsDialog = Elements.findById<HTMLDialogElement>(ElementIds.settingsDialog);
 
   static {
-    this.restartDialog.addEventListener('click', event => {
-      if (event.target === this.restartDialog) {
-        this.restartDialog.close('cancel');
+    for (const el of document.getElementsByClassName('modal-dialog')) {
+      const dialog = el as HTMLDialogElement;
+
+      dialog.addEventListener('click', this.clickOutsideDialogListener(dialog));
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)